main.py

```diff
@@ -0,0 +1,99 @@
+import os
+import datetime
+import pandas as pd
+import dxpy as dx
+
+from util import dx_login, send_mail
+
+
+seq = [x.upper() for x in os.environ['ENV_SEQ'].split(',')]
```

Comment:
any reason this isn't in `main()`? if it needs to be here it should be in capitals to conform with pep8 for constants

Reply:
yup will add in main()